qemu/tests/block_hotplug.py

```diff
@@ -121,42 +126,47 @@ def get_new_disk(disk1, disk2):
 
             device_list.append(device)
             if not new_disks:
-                raise exceptions.TestFail("Cannot find new disk after hotplug.")
+                test.fail("Cannot find new disk after hotplug.")
 
             if params.get("need_plug") == "yes":
                 disk = new_disks[0]
             else:
                 disk = new_disks[num]
 
+            session = vm.wait_for_login(timeout=timeout)
+            if params.get("format_disk", "no") == "yes":
+                if iteration == 0:
+                    error_context.context("Format disk", logging.info)
+                    utils_misc.format_windows_disk(session,
+                                                   disk_index[num],
+                                                   mountpoint=disk_letter[num])
             error_context.context("Check block device after hotplug.",
                                   logging.info)
-            if params.get("pci_test_cmd"):
+            if params.get("disk_op_cmd"):
                 if params.get("os_type") == "linux":
-                    test_cmd = params.get("pci_test_cmd") % (disk, disk)
+                    test_cmd = params.get("disk_op_cmd") % (disk, disk)
+                elif params.get("os_type") == "windows":
+                    test_cmd = params.get("iozone_cmd") % (disk_letter[num],
+                                                           disk_letter[num])
```

Comment:
Hi xuhan,

For this, it because linux and widows use totally different disk rw command,  If use the same parameter like disk_op_cmd (it define  for linux  rw test),  when for windows test, it need to replace the disk_op_cmd with windows uses (iozone_cmd). In order to easier the code, use 2 different param to run directly here~

Reply:
Logically speaking, it does the same thing for these two platforms: doing read/write to this disk. It's true that the two platform have different commands, and that is why we made it configurable.